tailwind.config.js

```diff
@@ -0,0 +1,172 @@
+module.exports = {
```

Comment:
did you get these configs from the Lucas's design?

Reply:
@tauantcamargo :point_up: ?